model-optimizer/extensions/ops/tensor_iterator.py

```diff
@@ -393,10 +385,12 @@ def check_field(record, field):
             if not is_fully_defined(in_rec_end) or not is_fully_defined(in_rec_start):
                 iterations_count = dynamic_dimension_value
                 continue
```

Comment:
just call `continue` in case of this condition.

Reply:
fixed